pkg/resources/fluentbit/configsecret.go

```diff
@@ -360,10 +360,11 @@ func (r *Reconciler) generateUpstreamNode(index int32) upstreamNode {
 	podName := r.Logging.QualifiedName(fmt.Sprintf("%s-%d", fluentd.ComponentFluentd, index))
 	return upstreamNode{
 		Name: podName,
-		Host: fmt.Sprintf("%s.%s.%s.svc.cluster.local",
+		Host: fmt.Sprintf("%s.%s.%s.svc.%s",
```

Comment:
I think this format string will have the same problem with the trailing dot if `ClusterDomain` is set to empty as I noted earlier with the chart template.